### PR TITLE
update html and react client to new header

### PR DESCRIPTION
**Pull request by mmacxm**
- add tenant title
- simplify user select with tenant groups

Screen Shot 2022-12-01 at 1 19 06 PM

### CODE REVIEW

File: client/react/src/components/Nav.jsx [hunk 1]
```diff
@@ -4,23 +4,29 @@ import {Types} from '../types'
...
-        { (ticketId || newTicket) &&
-          <>
-            <span>/</span>
-            <span>{ticketId ?? newTicket}</span>
-          </>
-        }
-        <span>|</span>
-        <span>{admin}</span>
+      <div>
+        <h1>{current.tenant}</h1>
```
Comment (johanfylling) [inline client/react/src/components/Nav.jsx:16]: If there is no `user` cookie present, I get a broken page and a bunch of

```
Uncaught TypeError: Cannot read properties of undefined (reading 'tenant')
    at Nav (Nav.jsx:16:1)
    at renderWithHooks (react-dom.development.js:16305:1)
    at mountIndeterminateComponent (react-dom.development.js:20074:1)
    at beginWork (react-dom.development.js:21587:1)
    at beginWork$1 (react-dom.development.js:27426:1)
    at performUnitOfWork (react-dom.development.js:26557:1)
    at workLoopSync (react-dom.development.js:26466:1)
    at renderRootSync (react-dom.development.js:26434:1)
    at recoverFromConcurrentError (react-dom.development.js:25850:1)
    at performConcurrentWorkOnRoot (react-dom.development.js:25750:1)
```

File: client/html/public/scripts/login.js [hunk 2]
```diff
@@ -10,31 +10,49 @@ function initialize() {
...
+  const [currentTenant, currentUser] = currentAccount.split('/').map((account) => account.trim());
+
   menu.innerHTML = `\
   <form id="login">
-    <label for="user-picker">Current tenant / user</label>
-    <select id="user-picker" name="user" class="user" onchange="login(true)">
-      ${userList.map((user) => 
-        `<option ${user == currentUser ? "selected" : ""}>${user}</option>`
+    <label for="account-select">User</label>
+    <select id="account-select" name="account" class="account" onchange="login(true)">
```
Comment (johanfylling) [inline client/html/public/scripts/login.js:35]: On Firefox, the drop-down is squished all the way to the right-hand side of the page. Might not be much we can do about it, but it looks less good than on Chrome, where there is a nice gap of air.

(Don't know how to make a screen grab, as the drop-down disappears whenever I start the `Screenshot` app 😅 )

**Review by johanfylling — APPROVED** [review]
LGTM 👍, except for the react app crashing when there is no `user` cookie 😄 .

Approving with the expectation that'll get fixed.

File: client/html/public/scripts/login.js (hunk 2 above)
Comment (mmacxm) [reply client/html/public/scripts/login.js:35]: looks fine on firefox?
Screen Shot 2022-12-02 at 11 45 01 AM

**Pull request merged**